common-arcana.lic

```diff
@@ -200,18 +200,22 @@ module DRCA
   end
 
   def find_cambrinth(cambrinth, stored_cambrinth, cambrinth_cap)
-    if stored_cambrinth
-      DRC.bput("get my #{cambrinth}", 'You get')
-    elsif DRSkill.getrank('Arcana').to_i < cambrinth_cap * 2 + 100
-      DRC.bput("remove my #{cambrinth}", 'You remove', 'You slide', 'You sling', 'You take', 'You detach')
+    if (DRC.right_hand || DRC.left_hand) !~ /#{cambrinth}/i
```

Comment:
💡 I think both of these would read better as guard clauses, something like 
```ruby
def find_cambrinth(cambrinth, stored_cambrinth, cambrinth_cap)
    return if ...

    if stored_cambrinth
    ...
```

Reply:
I can do that instead if you want.

Reply:
It's something to keep in mind for the future, e.g. the next time you're in this file. Doesn't need a whole PR just for that change.